<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_jedzenie">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jedzenie</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 255, 238);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_log">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>321</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Logowanie</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>251</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nazwa użytkownika:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nazwa">
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Hasło:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_haslo">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_nowe_konto">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Utwórz nowe konto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_zaloguj">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Zaloguj</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>340</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>*Hasło powinno zawierać co najmniej jedną literę małą oraz wielką, a także cyfrę</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="button_ng_pies">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PIES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_ng_kot">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>KOT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="button_wroc_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>12</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dostępne posiłki</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_dod_posilek">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dodaj własny posiłek</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_jedzenie">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>471</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dodaj menu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Podaj nick innego użytkownika,</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_dodaj_menu">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>200</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj menu innego użytkownika</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nick">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>170</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>130</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>aby dodać jego menu:</string>
      </property>
     </widget>
     <zorder>button_wroc_2</zorder>
     <zorder>label_8</zorder>
     <zorder>button_dod_posilek</zorder>
     <zorder>listWidget_jedzenie</zorder>
     <zorder>label_15</zorder>
     <zorder>button_dodaj_menu</zorder>
     <zorder>lineEdit_nick</zorder>
     <zorder>label_17</zorder>
     <zorder>label_16</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="button_wroc_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wybierz zabawę:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_zabawy">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>411</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>110</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>oraz czas jej trwania:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_czas_zabawy">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>150</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="button_wroc_4">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_czas_snu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>441</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wybierz czas snu przesuwając suwakiem</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2_czas_snu">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>140</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>150</y>
        <width>31</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_uspij">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Śpij</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="button_wroc_5">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wybierz doktora do którego chcesz pójść ze zwierzątkiem:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_chirurg">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>170</y>
        <width>181</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Chirurg</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_dermatolog">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Dermatolog </string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_radiolog">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Radiolog</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="button_wroc_6">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>581</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Dodaj posiłek</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>351</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Jest to posiłek czy przekąska?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox_5_posilek">
            <property name="text">
             <string>posiłek</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_4_przekaska">
            <property name="text">
             <string>przekąska</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Podaj nazwę posiłku (musi być unikalna!)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2_nazwapos"/>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Dla kogo jest przeznaczony posiłek</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_kot">
            <property name="text">
             <string>kot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3_pies">
            <property name="text">
             <string>pies</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2_wszystkie">
            <property name="text">
             <string>wszystkie</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Podaj kaloryczność posiłku (1-500)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="button_zatw_posilek">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>290</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ZATWIERDŹ</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>KONIEC GRY! </string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_nz_przegrana">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Nowe zwierzątko</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_nk_przegrana">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>230</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Zmień konto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_wynik">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Pokaż wynik</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_wynik1">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Twój wynik to</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_wynik3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>170</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_wynik2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>170</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>dni</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_zabawa">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zabawa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_wizyta">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wizyta u doktora</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zapisz">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ZAPISZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ZDROWIE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAMOPOCZUCIE:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DNI:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2_zdrowie">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3_samopoczucie">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <zorder>lcdNumber_3_samopoczucie</zorder>
   <zorder>lcdNumber_2_zdrowie</zorder>
   <zorder>graphicsView</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>label_7</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>button_wizyta</zorder>
   <zorder>button_sen</zorder>
   <zorder>button_zabawa</zorder>
   <zorder>button_jedzenie</zorder>
   <zorder>zapisz</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
